mix.exs

```diff
@@ -71,6 +71,10 @@ defmodule ExDoc.Mixfile do
           ExDoc.Markdown.Hoedown
         ],
 
+        "Custom Markdown Processors": [
```

Comment:
Let's nuke this since we should put it all on the `ExDoc.Markdown` module.

Reply:
As I said above, I quite like this for documentation purposes: It literally points to the reader how they can extend ExDoc with their own markdown processors.